docs/ref/contrib/admin/javascript.txt

```diff
@@ -0,0 +1,74 @@
+
+========================
+Admin javascript tweaks
+========================
+
+.. versionadded:: developement 
+
+Inline hooks
+------------
+
+Sometimes you need to execute some JavaScript or jQuery upon adding or removing inlines in the admin change_form.
+
+Django admin provide two jQuery events formset:added and formset:removed to do so.
+The event handler is passed three arguments event, $row, formsetName.
+event: is the ``jQuery`` event.
+$row: is the newly added (or removed) row.
+formsetName: the the formset the row belongs to.
+
+The event is fired using the ``django.jQuery`` namespace. See :doc:`/ref/contrib/admin/jQuery` for information.
```

Comment:
You need to add `.. _contrib-admin-jquery` above the jQuery heading on the other page, then use
```
:ref:`contrib-admin-jquery`
```
here.